go.mod

```diff
@@ -2,10 +2,11 @@ module github.com/cloudnativelabs/kube-router
 
 require (
 	github.com/aws/aws-sdk-go v1.38.70
-	github.com/containerd/containerd v1.5.2 // indirect
+	github.com/containerd/containerd v1.5.4 // indirect
```

Comment:
1.5.5 is out, do you want to update it to this version?

Reply:
Updated to 1.5.5